.circleci/config.yml

```diff
@@ -25,6 +28,20 @@ jobs:
       - run:
           name: Compile Cérès
           command: opam install ./coq-ceres.opam
+      - run:
+          name: Test dependants
+          command: |
+            PINS=$(opam list -s --pinned --columns=package | xargs | tr ' ' ,)
+            PACKAGES=`opam list -s --depends-on coq-ceres --coinstallable-with $PINS`
```

Comment:
Wouldn't it be simpler with something like `opam install $DEPS`?

Reply:
Not all DEPS are available on every version of Coq.

Reply:
Hmm seems already handled by the dependency-of-dependant-checking script.